test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -21,10 +21,40 @@ namespace Microsoft.ML.AutoML.Test
 {
     public class AutoFitTests : BaseTestClass
     {
+        // Marker necessary for AutoFitContextLogTest to ensure that the wanted logs
+        // from Experiment's sub MLContexts were relayed to the main calling MLContext.
+        bool _markerAutoFitContextLogTest;
         public AutoFitTests(ITestOutputHelper output) : base(output)
         {
         }
 
+        private void MlContextLog(object sender, LoggingEventArgs e)
+        {
+            // Log containing ImageClassificationTrainer will only come from AutoML's sub
+            // contexts.
+            if (!_markerAutoFitContextLogTest && e.Message.Contains("[Source=ImageClassificationTrainer;"))
+                _markerAutoFitContextLogTest = true;
+        }
+
+        [TensorFlowFact]
+        public void AutoFitContextLogTest()
+		{
```

Comment:
Tab characters used instead of spaces:
```suggestion
        public void AutoFitContextLogTest()
        {
```

Reply:
Thank you, my local VS keeps adding tabs, I have made the fix.